docs/pages/server-access/guides/host-user-creation.mdx

```diff
@@ -67,33 +92,33 @@ spec:
        "ALL = (root) NOPASSWD: /usr/bin/systemctl restart nginx.service"
     ]
     node_labels:
-      'env': 'devel'
-```
-Create the role:
-```code
-$ tctl create -f auto-users.yaml
-# role 'auto-users' has been created
+      "host_user_creation": "enabled"
 ```
 
-Each value of the `logins` field must conform to the username requirements
-of the Linux distribution being used. See [User/Group Name Syntax](https://systemd.io/USER_NAMES/) for requirements in common distributions.
+The `create_host_user_mode` field enables host user creation when the value is
+`keep`.  When a user with the `auto-users` role logs in to a server that matches
+the `host_user_creation:enabled` label, the Teleport SSH Service creates a host
+user, adds it to the groups listed in `host_groups`, and gives it the sudoer
+permissions specified in the `host_sudoers` field. In this case, the new user
+receives permission to restart the Nginx service as root.
 
-<Admonition type="warning">
+{/*TODO (ptgott): We should move the information below into a reference guide*/}
+<Details title="Customizing host user creation">
 
-When a Teleport user accesses an SSH Service instance, Teleport checks each of the
-user's roles that match the instance. If at least one role matches the instance
-but does not set `create_host_user_mode`, automatic user creation will be disabled.
-Roles that do not match the server will not be checked.
+`create_host_user_mode` can also be set to `insecure_drop`, which deletes users
+once the session ends. However, in this mode it is possible for a created user
+to get the same UID as a previously deleted user, which would give the new user
+access to all of the old user's files if they are not deleted. Use `keep` mode
+unless you really need users to be removed.
```

Comment:
Nit: I would move this note to the end of the section. We generally don't want people to use this mode as it comes with security risks.

Reply:
👍 Moved this to the end of the `Details` box, which is closed by default